src/plugins/goto/README.md

```diff
@@ -0,0 +1,24 @@
+Goto Plugin
+===========
+
+The goto plugin is a pre-stepleave plugin. It is executed before 
+`impress:stepleave` event, and will alter the destination where to transition next.
+
+Example:
+
+        <!-- When leaving this step, go directly to "step-5" -->
+        <div class="step" data-goto="step-5">
+
+        <!-- When leaving this step with next(), go directly to "step-5", instead of the next step.
+             If moving backwards to previous step - e.g. prev() instead of next() - then go to "step-1". -->
+        <div class="step" data-goto-next="step-5" data-goto-prev="step-1">
+
+        <!-- data-goto-key-list and data-goto-next-list allow you to build advanced non-linear navigation. -->
+        <div class="step" data-goto-key-list="ArrowUp ArrowDown ArrowRight ArrowLeft" data-goto-next-list="step-4 step-3 step-2 step-5">
```

Comment:
It's not clear that goto-key-list is a list of `KeyboardEvent.key` values. It would be good to mention what values can be used here for different keys - probably pointing to MDN documentation of `KeyboardEvent.key` would be enough.

Reply:
Good point.